pages/standards.tsx

```diff
@@ -111,6 +111,15 @@ const Standards = (data: StandardsProps) => {
                                         Imaging
                                     </Link>
                                 </li>
+                                <li>
+                                    <img
+                                        src="https://static.thenounproject.com/png/1167421-200.png"
+                                        width={20}
+                                    />
```

Comment:
I think this icon should come after the `Link`, not before it. Also we should try using a [font awesome icon](https://fontawesome.com/v4/icon/external-link) instead of an external png reference.

Reply:
Thank you for your feedback @onursumer ! Could you check again? I just updated the code based on your suggestion.